File: models/quotes/javascript/bubbleSort.js
function bubbleSort(array) {
	for (i = 0; i < array.length - 1; i++) {
		for (j = 0; j < array.length - 1; j++) {
    		if (array[j] > array[j + 1]) {
				let temp = array[j + 1];
				array[j + 1] = array[j];
				array[j] = temp;
			}
		}
	}
	return array;
}

File: models/quotes/python/bubbleSort.py
def bubbleSort(array):
	for i in range(0, len(array)-1, 1):
		for j in range(0, len(array)-1, 1):
   			if array[j] > array[j+1]:
 				temp = array[j+1]
 				array[j+1] = array[j]
 				array[j] = temp;

	return array